osp/views/question.py

```diff
@@ -1,71 +1,62 @@
-from rest_framework import viewsets, status
+from rest_framework import status, viewsets
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 
-from osp.models import (
-    UserInformation, Form, Question, Choice,
-    Checkbox, Dropdown, Paragraph, ShortAnswer,
-    Date, Time, FileUpload
-)
-from osp.serializers.question import QuestionSerializer
-from osp.serializers.fields import (
-    ChoiceSerializer, CheckboxSerializer, DropdownSerializer,
-    ParagraphSerializer, ShortAnswerSerializer, ParagraphSerializer,
-    DateSerializer, TimeSerializer, FileUploadSerializer
-)
```

Comment:
so a lot of these were unused right?

Reply:
Yes, @isabelcosta . I observed that most of them were copy-pasted from one file to another.